### ISSUE TITLE
Document alternative to  deprecated enable_cluster_monitoring in rancher 2.5

### ISSUE THREAD

**Issue by jmatsushita** [rancher/terraform-provider-rancher2#644]
Hello there,

Given that `Monitoring in Cluster Manager has been deprecated as of Rancher v2.5.0` I suppose that the recommended way to deploy monitoring is through a `rancher2_app_v2` instead of `enable_cluster_monitoring`? If so maybe it would be a nice addition to the documentation?

Cheers,

Jun

**Comment by rawmind0** [comment]
Hi @jmatsushita , the v1 monitoring is not deprecated. There is a limitation with new versions that is noted at provider docs, https://registry.terraform.io/providers/rancher/rancher2/latest/docs/resources/cluster#creating-rancher-v2-rke-cluster-enabling-and-customizing-monitoring , but both options are available . Also, at rancher docs both monitoring options are explained, https://rancher.com/docs/rancher/v2.x/en/cluster-admin/tools/

**Comment by jmatsushita** [comment]
I see, so this message 
![image](https://user-images.githubusercontent.com/356097/116555998-44d60100-a8fd-11eb-946b-f06957a1730a.png)

is only related to the Cluster Manager interface? When using `enable_cluster_monitoring` in Rancher 2.5 it will install the monitoring chart and make it available in the Cluster Explorer?

**Comment by rawmind0** [comment]
Nope, monitoring v1 is for cluster manager ui and monitoring v2 is for cluster explorer. We are migrating to v2 but v1 is still available

**Comment by jmatsushita** [comment]
Ok good to know. Not to be pendantic, but the message says that in Rancher 2.5 monitoring v1 is deprecated. It might still be possible to  use both v1 and v2, but I think that this terraform module documentation should reflect that v1 monitoring is deprecated (from 2.5). :)  For instance something like: 

> With rancher until 2.4, you can use the `enable_cluster_monitoring` and `cluster_monitoring_input` for v1 monitoring. However, from rancher 2.5, this method is deprecated in favor of using `rancher2_app_v2` terraform resource together with the `rancher-monitoring` operator helm chart (see [example usage](https://registry.terraform.io/providers/rancher/rancher2/latest/docs/resources/app_v2#example-usage)).

**Comment by jmatsushita** [comment]
These examples for `rancher2_app_v2` are more detailed and include logging and other rancher certified apps. https://github.com/rancher/terraform-provider-rancher2/blob/master/docs/guides/apps_marketplace.md

**Comment by matttrach** [comment]
This issue is a couple of years old now and can only reference unsupported versions, so I am going to close it, feel free to open a new issue if this is still an problem.
